osrsbox/items_api/item_equipment.py

```diff
@@ -0,0 +1,77 @@
+"""
+Author:  PH01L
+Email:   phoil@osrsbox.com
+Website: https://www.osrsbox.com
+
+Copyright (c) 2019, PH01L
+
+###############################################################################
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+###############################################################################
+"""
+
+import collections
+from typing import Dict
+from typing import List
+
+import safe_cast
+
+
+class ItemEquipment:
+    """This class defines the equipment structure and properties for an OSRS item."""
+    def __init__(self):
+        self.properties = [

```

Comment:
I think you should just turn this into object attributes.

Whats the benefit of keeping the properties like this? I don't think your setters are updating this structure or retrieving from this structure.